src/main/java/LoginPage.java

```diff
@@ -0,0 +1,35 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage {
+    private final WebDriver driver;
+    private final String loginPageURL = "https://mail.yandex.com/";
+    private final By enterButton = By.xpath("//*[@id=\"index-page-container\"]/div/div[2]/div/div/div[4]/a[2]");
+    private final By emailField = By.id("passp-field-login");
+    private final By loginButton = By.id("passp:sign-in");
+    private final By passwordField = By.id("passp-field-passwd");
+
+    public LoginPage(WebDriver driver) {
```

Comment:
You have singleton for WebDriver, there is no needs to pass driver as param

Reply:
This was changed